pkg/sfu/rtpmunger.go

```diff
@@ -148,16 +156,42 @@ func (r *RTPMunger) PacketDropped(extPkt *buffer.ExtPacket) {
 	}
 
 	r.extLastSN = r.extSecondLastSN
+	r.updateSnOffset()
 }
 
 func (r *RTPMunger) UpdateAndGetSnTs(extPkt *buffer.ExtPacket) (*TranslationParamsRTP, error) {
 	diff := int64(extPkt.ExtSequenceNumber - r.extHighestIncomingSN)
+	if (diff == 1 && len(extPkt.Packet.Payload) != 0) || diff > 1 {
```

Comment:
Re-arranging this method to check for fast path check as the first one. This should be the path taken almost all the time. So, avoiding other checks and falling back to this default should help a bit.